modbus_south.cpp

```diff
@@ -1498,6 +1498,9 @@ ModbusCacheManager	*manager = ModbusCacheManager::getModbusCacheManager();
 		{
 			return NULL;
 		}
+		
+		Logger::getLogger()->error("Check 1:regValue is %d", regValue);
```

Comment:
Replace with Logger::getLogger()->debug

Reply:
Sorry, this should have been removed. I have corrected it now.